src/configurations/destinations/comscore/db-config.json

```diff
@@ -12,41 +12,59 @@
       "blacklistedEvents",
       "whitelistedEvents",
       "eventFilteringOption",
+      "fieldMapping",
       "oneTrustCookieCategories",
       "consentManagement",
       "ketchConsentPurposes"
     ],
     "excludeKeys": [],
-    "supportedSourceTypes": ["android", "ios"],
+    "supportedSourceTypes": ["android", "ios", "web"],
     "supportedMessageTypes": {
       "device": {
         "android": ["identify", "track", "screen"],
-        "ios": ["identify", "track", "screen"]
+        "ios": ["identify", "track", "screen"],
+        "web": ["page"]
       }
     },
     "supportedConnectionModes": {
       "android": ["device"],
-      "ios": ["device"]
+      "ios": ["device"],
+      "web": ["cloud"]
     },
     "destConfig": {
       "defaultConfig": [
-        "publisherId",
+        "publisherId"
+      ],
+      "android": [
         "appName",
```

Comment:
Is this change expected ? Making appName & rest of the fields source-type specific instead of generic configuration across sources ?

Reply:
this kind of change will need configuration migration as the structure of field changes when it's moved from defaultConfig to sourceType specific config cc: @sandeepdsvs